phpstan.neon.dist

```diff
@@ -0,0 +1,7 @@
+parameters:
+  bootstrapFiles:
+    - tests/bootstrap.php
+  paths:
+    - src
+    - tests
+  level: 0
```

Comment:
This is the default and most basic level. I see that this level already highlighted a few issues in test cases.

I'd be curious to see if we can raise the bar a little bit. Levels are 0-9, but I think attempting a level beyond 6 on an existing and complex project could turn out to be a nightmare.

Would you be willing to try and experiment a little with levels 1, 2, 3, …? I don't expect a level increase within this PR, of course, but just to have an idea of the amount of work needed…

Reply:
I would give it a try (a deeper level change) in a separate PR